src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/HybridSharedTableBuilder.java

```diff
@@ -105,7 +106,8 @@ public HybridSharedTableBuilder withPollIntervalSeconds(Integer pollIntervalSeco
         return this;
     }
 
-    private class IteratingCreateTableRequestFactory implements CreateTableRequestFactory { // TODO add unit test
+    @VisibleForTesting
+    static class IteratingCreateTableRequestFactory implements CreateTableRequestFactory {
```

Comment:
and while I still like CreateTableRequestFactoryEnsemble better (over IteratingCreateTableRequestFactory), CreateTableRequestFactoryFamily could work too. (one def. for "ensemble": "A group of separate things that contribute to a coordinated whole.")